reddit_liveupdate/hooks.py

```diff
@@ -0,0 +1,35 @@
+from pylons import c
+
+from r2.config import feature
+from r2.lib.hooks import HookRegistrar
+from r2.models import DefaultSR, NamedGlobals, NotFound
+
+from reddit_liveupdate.models import LiveUpdateEvent
+from reddit_liveupdate.pages import LiveUpdateHappeningNowBar
+
+hooks = HookRegistrar()
+
+
+@hooks.on("hot.get_content")
+def add_featured(controller):
+    if not feature.is_enabled('live_happening_now'):
+        return None
+
+    # Not on front page
+    if not isinstance(c.site, DefaultSR):
+        return None
+
+    # Not on first page of front page
+    if getattr(controller, 'listing_obj') and controller.listing_obj.prev:
+        return None
+
+    event_id = NamedGlobals.get('live_happening_now')
```

Comment:
This will throw a `NotFoundException` if it hasn't been set, unless you pass a second parameter as the default.  That'll only happen the very first time (assuming we set it to `None` on clear, rather than deleting the key), but probably should still do it.